install_config/aggregate_logging.adoc

```diff
@@ -1462,6 +1462,149 @@ have the same multi-tenant capabilities and your data will not be restricted by
 user access to a particular project.
 ====
 
+[[sending-logs-to-external-rsyslog]]
+== Sending Logs to an External Rsyslog Instance
+
+Use the `fluent-plugin-remote-syslog` plug-in on the host to send logs to an external rsyslog server. 
```

Comment:
The plugin is references "syslog", but am I ok referencing rsyslog here?

Reply:
I think they're both compatible, but I was saying "syslog" to keep it in line with what the gem said.